ocs_ci/helpers/keyrotation_helper.py

```diff
@@ -408,3 +408,120 @@ def wait_till_keyrotation(self, device_handle):
             assert False
 
         return True
+
+
+def enable_key_rotation():
+    """
+    Enable Key rotation and verify its status for Noobaa and Storage cluster.
+
+    """
+    osd_keyrotation = OSDKeyrotation()
+    noobaa_keyrotation = NoobaaKeyrotation()
+    osd_keyrotation.enable_keyrotation()
+    noobaa_keyrotation.enable_keyrotation()
+
+    assert (
+        osd_keyrotation.is_keyrotation_enable()
+    ), "Encryption Key rotation is not enabled for OSDs"
+    assert (
+        noobaa_keyrotation.is_keyrotation_enable
+    ), "Keyrotation is not enabled in the storagecluster object."
+    assert (
+        noobaa_keyrotation.is_noobaa_keyrotation_enable
+    ), "Keyrotation is not enabled in the noobaa object."
+
+
+def set_key_rotation_time(value):
```

Comment:
isn't this same as https://github.com/red-hat-storage/ocs-ci/blob/c225623e6f8411bbdac677326cfe91971ce46023/ocs_ci/helpers/keyrotation_helper.py#L86